api4/plugin_test.go

```diff
@@ -1702,34 +1723,58 @@ func TestInstallMarketplacePlugin(t *testing.T) {
 				*cfg.PluginSettings.AllowInsecureDownloadURL = true
 			})
 
-			pRequest = &model.InstallMarketplacePluginRequest{Id: "testplugin2"}
-			manifest2, _, err := client.InstallMarketplacePlugin(pRequest)
-			require.NoError(t, err)
-			require.NotNil(t, manifest2)
-			require.Equal(t, "testplugin2", manifest2.Id)
-			require.Equal(t, "1.2.3", manifest2.Version)
+			t.Run("Install prepackaged, not listed plugin with Marketplace enabled", func(t *testing.T) {
```

Comment:
It's hard to read in the diff: This is the test added to cover the changes of the PR.